tests/foreman/api/test_user.py

```diff
@@ -55,28 +62,7 @@ def test_create(self, attrs):
             self.assertIn(name, real_attrs.keys())
             self.assertEqual(value, real_attrs[name])
 
-    @decorators.data(
-        {u'admin': False},
-        {u'admin': True},
-        {u'firstname': gen_string('alphanumeric', randint(1, 50))},
-        {u'firstname': gen_string('alpha', randint(1, 50))},
-        {u'firstname': gen_string('cjk', randint(1, 50))},
-        {u'firstname': gen_string('latin1', randint(1, 50))},
-        {u'firstname': gen_string('numeric', randint(1, 50))},
-        {u'firstname': gen_string('utf8', randint(1, 16))},
-        {u'lastname': gen_string('alphanumeric', randint(1, 50))},
-        {u'lastname': gen_string('alpha', randint(1, 50))},
-        {u'lastname': gen_string('cjk', randint(1, 50))},
-        {u'lastname': gen_string('latin1', randint(1, 50))},
-        {u'lastname': gen_string('numeric', randint(1, 50))},
-        {u'lastname': gen_string('utf8', randint(1, 16))},
-        {u'login': gen_string('alphanumeric', randint(1, 100))},
-        {u'login': gen_string('alpha', randint(1, 100))},
-        {u'login': gen_string('cjk', randint(1, 100))},
-        {u'login': gen_string('latin1', randint(1, 100))},
-        {u'login': gen_string('numeric', randint(1, 100))},
-        {u'login': gen_string('utf8', randint(1, 33))},
-    )
+    @decorators.data(*_user_attrs())
```

Comment:
I believe the real reason behind the number of data points passed to ``test_delete`` is to make sure that, no matter what type of data is used, you should be able to delete it. Using ``_user_attrs()`` won't provide the same data set :/